docs/require-login.md

```diff
@@ -15,6 +15,36 @@ Requiring login on individual sites is as easy as unchecking the site's public s
 
 ## Excluding Pages and Endpoints
 
+There are 2 ways to exclude pages and endpoints from requiring login, using the `bypass-tokens` config option or the `hm-require-login.allowed_pages` filter.
+
+### Bypass Tokens
+
+Bypass tokens are a way to allow tools like Lighthouse or Qualys access to development or staging environments using a unique query string token. This lets you check performance insights on non-production environments for non-logged in users so you can see the experience most site visitors will get.
```

Comment:
Not sure what "Qualys" is here, is that referencing SSL Labs? If so, authentication isn't necessary for that in any case.